bitnami/keycloak/values.yaml

```diff
@@ -42,6 +42,20 @@ enableServiceLinks: true
 ## @param commonAnnotations Annotations to add to all deployed objects
 ##
 commonAnnotations: {}
+## @param dnsPolicy DNS Policy for pod
+## ref: https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/
+## E.g.
+## dnsPolicy: ClusterFirst
+dnsPolicy: ""
+## @param dnsConfig DNS Configuration pod
```

Comment:
```suggestion
## @param dnsConfig.ndots DNS Configuration pod
```

Reply:
> dnsConfig.ndots

sorry I committed that by mistake while testing, just update like this, dnsConfig: {} & so ndots in the comments are no more required.